components/CountryEditForm/index.js

```diff
@@ -0,0 +1,105 @@
+import Link from "next/link";
+import styled from "styled-components";
+
+export default function CountryEditForm({
+  country,
+  onSubmitEdit,
+  handleInputChangeName,
+  onEdit,
+  handleInputChangeStartDate,
+  handleInputChangeEndDate,
+}) {
+  if (country === undefined) {
+    return null;
+  }
+  return (
+    <>
+      <form onSubmit={onSubmitEdit}>
+        <StyledFieldset>
+          <StyledDiv>
+            <StyledLabel htmlFor="name">Land hinzufügen: </StyledLabel>
+            <StyledInput
+              value={country.name}
+              maxLength="60"
+              name="name"
+              id="name"
+              type="text"
+              placeholder="Österreich, Australien..."
```

Comment:
You already set the value of this input field to country.name, which is a good idea, because it's very comfortable for the user if they don't want to change everything. But then you don't need the placeholder.